Content Expansion: New Alien Faction: The Gegno (#6454)
endless-sky/endless-sky: data/gegno/ships.txt

# Copyright (c) 2021 by Saugia
#
# Endless Sky is free software: you can redistribute it and/or modify it under the
# terms of the GNU General Public License as published by the Free Software
# Foundation, either version 3 of the License, or (at your option) any later version.
#
# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
# PARTICULAR PURPOSE.  See the GNU General Public License for more details.

ship Shale
	sprite ship/shale
	thumbnail thumbnail/shale
	swizzle 0
	attributes
		category Transport
		cost 180000
		mass 121
		bunks 7
		"cargo space" 36
		drag 1.7
		"engine capacity" 58
		"fuel capacity" 400
		"heat dissipation" 0.8
		hull 825
		"outfit space" 135
		"required crew" 1
		shields 0
		"weapon capacity" 12
		weapon
			"blast radius" 18
			"shield damage" 40
			"hull damage" 132
			"hit force" 210
	outfits
		"C3 Scrap Cell"
		Hyperdrive
		"R01 Skirmish Battery"
		"RG15 Torch Steering"
		"RG15 Torch Thruster"
	engine 0 40
	leak leak 50 60
	explode "small explosion" 5
	explode "tiny explosion" 10

> 	description "Due to the Gegno splitting up and colonizing new worlds, they needed a smaller ship to do a lot of the grunt work instead of always relying on their enormous (and expensive) City-Ships. The Shale is the Gegno's design of a shuttle-ship, small and compact with the single purpose as a transport vessel, whether it be cargo, trade goods, travellers, or recruits to move off homeworld to pursue the efforts of the Vi and the Scin."

Author: A dash makes more sense here than a colon. A list typically follows a colon, and a dash marks a next iteration of a phrase (if that makes sense), so having the "have all been carried..." part after a colon seems kind of run-on-y. Either way the suggestion works out.

> 	description "Due to the Gegno splitting up and colonizing new worlds, they needed a smaller ship to do a lot of the grunt work instead of always relying on their enormous (and expensive) City-Ships. The Shale is the Gegno's design of a shuttle-ship, small and compact with the single purpose as a transport vessel, whether it be cargo, trade goods, travellers, or recruits to move off homeworld to pursue the efforts of the Vi and the Scin."

Reviewer: How about this?
```suggestion
	description "Due to the Gegno splitting up and colonizing new worlds, they needed a smaller ship to do a lot of the grunt work instead of always relying on their enormous (and expensive) City-Ships. The Shale is the Gegno's design of a shuttle-ship, small and compact with the single purpose as a transport vessel. What exactly it transports varies widely - cargo, trade goods, travellers, or recruits for the war efforts of the Vi and the Scin have all been carried by this versatile ship.""
```

ship Hornfel
	sprite ship/hornfel
	thumbnail thumbnail/hornfel
	swizzle 0
	attributes
		category "Medium Warship"
		cost 1960000
		mass 740
		bunks 74
		"cargo space" 120
		drag 5.9
		"engine capacity" 94
		"fuel capacity" 400
		"heat dissipation" 0.7
		hull 14550
		"outfit space" 428
		"required crew" 58
		shields 0
		"weapon capacity" 140
		weapon
			"blast radius" 90
			"shield damage" 600
			"hull damage" 840
			"hit force" 900
	outfits
		"C17 Warzone Core"
		"Chloeric Turret" 2
		Hyperdrive
		"Irate Cannon" 4
		"R03 Warforge Battery"
		"RG18 Torch Steering"
		"RG18 Torch Thruster"
	engine -16.5 206
	engine 16.5 206
	gun -13.5 -228 "Irate Cannon"
	gun 13.5 -228 "Irate Cannon"
	gun -22.5 -204 "Irate Cannon"
	gun 22.5 -204 "Irate Cannon"
	turret 0 -112 "Chloeric Turret"
	turret 0 -41.5 "Chloeric Turret"
	leak leak 60 50
	leak flame 60 80
	explode "large explosion" 10
	explode "medium explosion" 25
	explode "small explosion" 25
	explode "tiny explosion" 10
	"final explode" "final explosion medium"

> 	description "Hornfels are the powerhorse of the Gegno Vi's military. They're decently sized, decently fast, and armed more than decently to be able to fight on their own. Because the Gegno Vi are always requiring more replacement supplies in war, they double as a secondary cargo transport as well, and are seen occasionally with cargo pods attached to the several racks lining the sides of the ship."

Reviewer: ```suggestion
	description "Hornfels are the workhorse of the Gegno Vi's military. They're decently sized, decently fast, and more than decently armed in order to be able to fight on their own. Because the Gegno Vi always require more replacement supplies in war, they double as a secondary cargo transport as well, and are seen occasionally with cargo pods attached to the several racks lining the sides of the ship."
```

ship Protolith
	sprite ship/protolith
	thumbnail thumbnail/protolith
	swizzle 0
	attributes
		category "Heavy Freighter"
		cost 17600000
		mass 1260
		bunks 292
		"cargo space" 530
		drag 16.1
		"engine capacity" 180
		"fuel capacity" 700
		"heat dissipation" 0.4
		hull 19575
		"outfit space" 768
		"required crew" 84
		shields 0
		"weapon capacity" 300
		weapon
			"blast radius" 280
			"shield damage" 1550
			"hull damage" 2050
			"hit force" 3950
	outfits
		"C27 Campaign Core"
		"Chloeric Turret" 2
		Hyperdrive
		"Irate Cannon" 2
		"R02 Battlezone Battery"
		"R03 Warforge Battery"
		"RG3 Torch Steering"
		"RG3 Torch Thruster"
	engine 0 250
	engine -41.5 242
	engine 41.5 242
	gun 37 -212 "Irate Cannon"
	gun 45 -199.5 "Irate Cannon"
	turret 0 -33 "Chloeric Turret"
	turret 0 8 "Chloeric Turret"
	leak leak 30 50
	leak flame 30 80
	leak "big leak" 50 30
	explode "huge explosion" 30
	explode "large explosion" 45
	explode "medium explosion" 35
	explode "small explosion" 25
	"final explode" "final explosion large"

> 	description "Protoliths are descendents of the Gegno's early city ships used to colonize the nearby planets. They carry hundreds of battle-ready Vi both in and out of war, and carry huge cargo loads to supply depots. They are not typically used as warships but can be mounted with a small payload that can threaten most ships smaller than them, or fend off larger ships long enough for reinforcements to arrive."

Reviewer: ```suggestion
	description "Protoliths are descendents of the Gegno's early city ships used to colonize nearby planets. They now carry hundreds of battle-ready Vi both in and out of war, and carry huge cargo loads to supply depots. They are not typically used as warships but can be mounted with a small payload that can threaten most ships smaller than them, or fend off larger ships long enough for reinforcements to arrive."
```

ship Halite
	sprite ship/halite
	thumbnail thumbnail/halite
	swizzle 0
	attributes
		category Fighter
		cost 129000
		mass 30
		bunks 1
		"cargo space" 1
		drag 0.74
		"engine capacity" 30
		"heat dissipation" 1
		hull 860
		"outfit space" 120
		"required crew" 1
		shields 0
		"weapon capacity" 30
		weapon
			"blast radius" 14
			"shield damage" 160
			"hull damage" 85
			"hit force" 160
	outfits
		"SC-1 Plasma Engines"
		"I70 Ameliorate Cell"
		"Talc Laser" 2
		
	engine -9 42
	engine 9 42
	gun -18.5 -44.5 "Talc Laser"
	gun 18.5 -44.5 "Talc Laser"
	leak leak 60 50
	explode "small explosion" 8
	explode "tiny explosion" 10

> 	description "Halites are a double-first for the Gegno. They are the Gegno Scin's newest ship design, and are the Gegno's first fighter-based craft. They are also the first Gegno ships equipped with early-stage beam weapon technology, with almost half the hull dedicated to it. Able to fly around the Gegno Vi's slower projectile techology, Halites are able to bring a swarm of instant damage to the slow-moving Vi warships, making them hard to deal with in numbers."

Reviewer: What do you mean by the `almost half the hull dedicated to it`? I wouldn't think that the hull mattered much when considering what guns are equipped.

Author: It's more of a visual design reference than an actual game mechanic reference- the model of the Halite has it so that the two gun mounts are in fact the lasers themselves tied into the frame, and the Talc Laser in the outfitter is what you see on the Halite itself.

Reviewer: Maybe this would make it clearer?
```suggestion
	description "Halites are a double-first for the Gegno. They are the Gegno Scin's newest ship design, and are the Gegno's first fighter-based craft. They are also the first Gegno ships equipped with early-stage beam weapon technology, with almost half the hull dedicated to holding two Talc Lasers. Able to fly around the Gegno Vi's slower projectile techology, Halites are able to bring a swarm of instant damage to the slow-moving Vi warships, making them hard to deal with in numbers."
```

ship Mica
	sprite ship/mica
	thumbnail thumbnail/mica
	swizzle 0
	attributes
		category Transport
		cost 180000
		mass 115
		bunks 9
		"cargo space" 36
		drag 1.7
		"engine capacity" 58
		"fuel capacity" 400
		"heat dissipation" 0.8
		hull 700
		"outfit space" 150
		"required crew" 1
		shields 0
		"weapon capacity" 10
		weapon
			"blast radius" 18
			"shield damage" 40
			"hull damage" 132
			"hit force" 210
	outfits
		"C3 Scrap Cell"
		Hyperdrive
		"R01 Skirmish Battery"
		"SC-12 Plasma Steering"
		"SC-12 Plasma Thruster"
	engine 0 40
	leak leak 30 40
	explode "small explosion" 4
	explode "tiny explosion" 12

> 	description "The Mica is the Gegno Scin's counterpart to a shuttlecraft, and is a derivative of the Gegno's Shale and the Vi's Slate. It is ever so slightly smaller for the slightly shorter Scin, but carries newer engine systems that almost double the speed of either the Shale, or the Gegno Vi's Slate. Combined with the speed and the room for a few more Scin, it is the relatively faster option of a shuttlecraft in Gegno space."

Reviewer: ```suggestion
	description "The Mica is the Gegno Scin's counterpart to a shuttlecraft, and is a derivative of the Gegno's Shale and the Vi's Slate. It is ever so slightly smaller for the slightly shorter Scin, but carries newer engine systems that almost double the speed of both the Shale and the Slate. Combined with the speed and the room for a few more Scin, it is a relatively faster option for a captain looking for a shuttlecraft in Gegno space."
```

ship Coesite "Coesite P"
	sprite ship/coesitep
	thumbnail thumbnail/coesitep
	add attributes
		mass 17
		drag 0.9
		"solar collection" 0.86
	outfits
		"Astuit Battery" 2
		"C7 Brawl Core"
		"Guile Anti-Missile"
		Hyperdrive
		"R02 Battlezone Battery"
		"SC-12 Plasma Thruster"
		"SC-14 Plasma Steering"

ship Felsic
	sprite ship/felsic
	thumbnail thumbnail/felsic
	swizzle 0
	attributes
		category "Light Freighter"
		cost 1960000
		mass 565
		bunks 78
		"cargo space" 160
		drag 5.9
		"engine capacity" 126
		"fuel capacity" 500
		"heat dissipation" 0.8
		hull 11000
		"outfit space" 300
		"required crew" 38
		shields 0
		"weapon capacity" 80
		weapon
			"blast radius" 74
			"shield damage" 340
			"hull damage" 650
			"hit force" 690
	outfits
		"Astuit Battery"
		"C7 Brawl Core"
		"Guile Anti-Missile"
		Hyperdrive
		"R02 Battlezone Battery"
		"SC-14 Plasma Steering"
		"SC-14 Plasma Thruster"
	engine 0 135
	turret 0 63 "Guile Anti-Missile"
	turret 0 -79.5 "Astuit Battery"
	leak leak 60 50
	leak flame 60 80
	explode "large explosion" 7
	explode "medium explosion" 20
	explode "small explosion" 28
	explode "tiny explosion" 16
	"final explode" "final explosion medium"

> 	description "Felsics are essentially stripped Schists repurposed for a more engaging role. In place of the more heavy engines that the Vi had mounted on, they have more efficient engines, dedicating the remaining space to a larger cargo capacity. The Scin also outfitted them with two turrent mounts, allowing them to double as a secondary combat vessel, and an extra hand in anti-shell duty when needed."

Reviewer: ```suggestion
	description "Felsics are essentially stripped Schists repurposed for a more engaging role. In place of the heavier engines that the Vi had mounted on, they have more efficient engines, dedicating the remaining space to a larger cargo capacity. The Scin also outfitted them with two turrent mounts, allowing them to double as a secondary combat vessel, and an extra hand in anti-shell duty when needed."
```

		mass 1146
		bunks 192
		"cargo space" 82
		drag 15
		"engine capacity" 182
		"fuel capacity" 600
		"heat dissipation" 1.4
		hull 18800
		"outfit space" 866
		"required crew" 133
		shields 0
		"weapon capacity" 212
		weapon
			"blast radius" 180
			"shield damage" 1650
			"hull damage" 1200
			"hit force" 1000
	outfits
		"Acuit Artillery" 2
		"C27 Campaign Core"
		Hyperdrive
		"R03 Warforge Battery"
		"SC-14 Plasma Steering"
		"SC-15 Plasma Engines"
	engine 28.5 196.5
	engine -28.5 196.5
	"reverse engine" -34.5 -186.5
		angle -360
	"reverse engine" 34.5 -186.5
		angle -360
	"steering engine" 106.5 31.5
		angle -63.001099
		left
	"steering engine" -106.5 31.5
		angle 63.001099
		right
	turret -72.5 17 "Acuit Artillery"
	turret 72.5 17 "Acuit Artillery"
	bay Fighter -71 -171.5
		over
	bay Fighter 71 -171.5
		over
	bay Fighter -69 -97
		over
	bay Fighter 69 -97
		over
	bay Fighter -65.5 120
		angle -180
		over
	bay Fighter 65.5 120
		angle -180
		over
	leak leak 50 50
	leak flame 80 140
	explode "huge explosion" 18
	explode "large explosion" 30
	explode "medium explosion" 46
	explode "small explosion" 50
	explode "tiny explosion" 40
	"final explode" "final explosion large"

> 	description "The mighty Felspar is a Scin creation to behold. As a testbed for the new Halite Fighters, Feldspars bring their own small fleets with them and overwhelm their enemy. They also double as a semi-experimental platform, carrying a brand new multi-engine system and the latest in Gegno plasma technology. Able to keep a decent stance away from a battle while still putting down fire, Feldspars are quite a dangerous well-rounded ship for their sizem but are susceptible to hull damage as any other Scin ship."

Reviewer: ```suggestion
	description "The mighty Felspar is a Scin creation to behold. As a testbed for the new Halite Fighters, Feldspars bring their own small fleets with them and overwhelm their enemy. They also double as a semi-experimental platform, carrying a brand new multi-engine system and the latest in Gegno plasma technology. Able to keep a decent stance away from a battle while still putting down fire, Feldspars are quite a dangerous, well-rounded ship for their size but are susceptible to hull damage as any other Scin ship."
```
